build.yaml

```diff
@@ -2042,6 +2042,19 @@ steps:
     - batch_database
     - deploy_auth
     - create_certs
+ - kind: deploy
```

Comment:
Was this duplicated now? I don't see the deletion of the previous code on the left.